projects/questionset-editor-library/src/lib/services/editor/editor.service.ts

```diff
@@ -249,7 +249,33 @@ export class EditorService {
     return this.publicDataService.post(option);
   }
 
-  submitRequestChanges(contentId, comment) {
+  updateComment(contentId,comment): Observable<any> {
+    const url = this.configService.urlConFig.URLS[this.editorConfig.config.objectType];
+    const option = {
+      url: url.UPDATE_COMMENT+contentId,
+      data: {
+        request: {
+          comments : [
+            {
+              comment : comment
```

Comment:
Why are not passing the "identifier" here. So that mulitple comments can go to the contentId/Question-set lelvel.
`[
{
  "identifier": "question-set_id"
   "comment" : "Question-set-comment1"
},
{
  "identifier": "question-set_id"
   "comment" : "Question-set-comment2"
}
]`


Also it enable to send comments for individual questions as well. 

`[
{
  "identifier": "question-set_id"
   "comment" : "Question-set-comment1"
},
{
  "identifier": "question-set_id"
   "comment" : "Question-set-comment2"
},
{
  "identifier": "question_id"
   "comment" : "Question1-comment1"
}
...
]`

Reply:
Right now the API only accepts only one question set comments and question set identifier will be taken from API params. example added below:
URL : <Base_URL>/questionset/v2/comment/update/<Questionset_ID>
1. Right now we don't have option for particular question level comments, Now we limited to all the comments write and read in one place, there we can put all the comments for each questions and modify accordingly.

Kindly let me know, if i'm not explained clear on this.